.pre-commit-config.yaml

```diff
@@ -2,22 +2,6 @@
 exclude: "migrations"
 
 repos:
-  - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.0
-    hooks:
-      - id: pyupgrade
-        args: ["--py37-plus"]
```

Comment:
Why remove pyupgrade?

Reply:
It is built into ruff... https://docs.astral.sh/ruff/rules/#pyupgrade-up